rel/files/app.config

```diff
@@ -1,4 +1,34 @@
 [
+ {kernel, [
+  {logger_level, warning},
+  {logger, [
+    %% Console logger
+    {handler, default, logger_std_h, #{}},
+    %% Disk logger for errors
+    {handler, disk_log, logger_disk_log_h,
+       #{config => #{
+           file => "{{mongooseim_log_dir}}/ejabberd.log",
+           type => wrap,
+           max_no_files => 5,
+           max_no_bytes => 2097152,
+           sync_mode_qlen => 2000,
+           drop_mode_qlen => 2000,
```

Comment:
How exactly this work? Seeing them set to the same value I can think that log messages will be dropped the moment logging switches to the sync mode.

Reply:
Ok, I just learned that from your doc:
> If sync_mode_qlen is set to the same value as drop_mode_qlen, synchronous mode is disabled. That is, the handler always runs in asynchronous mode, unless dropping or flushing is invoked.

I think this info could be added as a comment to the logger configuration.